convene-web/app/javascript/src/custom_components/button.scss

```diff
@@ -8,4 +8,10 @@
   .button-blue:hover {
     @apply bg-blue-700;
   }
+  .button-gray {
+    @apply bg-gray-500 text-white;
+  }
+  .button-gray:hover {
+    @apply bg-gray-700;
+  }
```

Comment:
Not blocking: I am curious what the semantics of these buttons are; and hope

Generally, there are ~4 different "actions" where pressing a button:

1. Changes nothing, and is purely for acknowledging information; like dismissing an alert.
2. Moves someone forward (or backward) in a workflow
3. Changes something safely (i.e. changes a value that can be easily changed back).
4. Changes something _dangerously_

I would prefer we use the `--modifier` syntax for denoting these things; i.e. `button --acknowledge` or `button --nav`  or `button --commit` or `button --commit-danger` 

This way when we build features, we're consistently signaling to the user what they can expect to happen when they push a button.